packages/sources/coinmetrics/test/integration/adapter.test.ts

```diff
@@ -1,4 +1,8 @@
-import { mockCoinmetricsResponseSuccess, mockCoinmetricsResponseSuccess2 } from './fixtures'
+import {
+  mockCoinmetricsRealizedVolResponseSuccess,
+  mockCoinmetricsResponseSuccess,
+  mockCoinmetricsResponseSuccess2,
```

Comment:
i know it's not your change but... lol

Related: I've seen both, but is there a best practice between having separate nock responses and having them all in a single chain of responses in `fixtures.ts`? 
ie: 
```javascript
{
   nock (<url>, {...
   }
   .get(...).quote(...).reply(...)
   .get(...).quote(...).reply(...)
}
```

Reply:
I'm not sure if there's a best practice!

I believe you can only have one Nock instance per URL, so if you need to have multiple mocks at once you should chain them in a single instance. However if you don't need the other mocks, you can split them out to ensure that only the mocks you're interested in are being applied.